includes/fraud-prevention/class-order-fraud-and-risk-meta-box.php

```diff
@@ -0,0 +1,129 @@
+		$wc_screen_id = \wc_get_page_screen_id( 'shop-order' );
+
+		add_meta_box( 'wcpay_order_fraud_and_risk_meta_box', __( 'Fraud &amp; Risk', 'woocommerce-payments' ), [ $this, 'display_order_fraud_and_risk_meta_box_message' ], $wc_screen_id, 'side', 'default' );
+	}
+
+	/**
+	 * Displays the contents of the Fraud & Risk meta box.
+	 *
+	 * @param \WC_Order $order The order we are working with.
+	 *
+	 * @return void
+	 */
+	public function display_order_fraud_and_risk_meta_box_message( $order ) {
+		$order = wc_get_order( $order );
+
+		if ( ! $order ) {
+			return;
+		}
+
+		$intent_id      = $this->order_service->get_intent_id_for_order( $order );
+		$charge_id      = $this->order_service->get_charge_id_for_order( $order );
+		$outcome_status = $this->order_service->get_fraud_outcome_status_for_order( $order );
+
+		if ( 'woocommerce_payments' !== $order->get_payment_method() ) {
+			$outcome_status = 'not_wcpay';
+		}
+
+		switch ( $outcome_status ) {
+			case Fraud_Outcome_Status::ALLOW:
+				$icon_url    = plugins_url( 'assets/images/icons/check-green.svg', WCPAY_PLUGIN_FILE );
+				$icon_alt    = __( 'Green check mark', 'woocommerce-payments' );
+				$status      = __( 'No action taken', 'woocommerce-payments' );
+				$description = __( 'The payment for this order passed your risk filtering.', 'woocommerce-payments' );
+				echo '<p class="wcpay-fraud-risk-meta-allow"><img src="' . esc_url( $icon_url ) . '" alt="' . esc_html( $icon_alt ) . '"> ' . esc_html( $status ) . '</p><p>' . esc_html( $description ) . '</p>';
+				break;
+
+			case Fraud_Outcome_Status::REVIEW:
+				$icon_url        = plugins_url( 'assets/images/icons/shield-stroke-orange.svg', WCPAY_PLUGIN_FILE );
+				$icon_alt        = __( 'Orange shield outline', 'woocommerce-payments' );
+				$status          = __( 'Held for review', 'woocommerce-payments' );
+				$description     = __( 'The payment for this order was held for review by your risk filtering. You can review the details and determine whether to approve or block the payment.', 'woocommerce-payments' );
+				$callout         = __( 'Review payment', 'woocommerce-payments' );
+				$transaction_url = WC_Payments_Utils::compose_transaction_url( $intent_id, $charge_id );
+				echo '<p class="wcpay-fraud-risk-meta-review"><img src="' . esc_url( $icon_url ) . '" alt="' . esc_html( $icon_alt ) . '"> ' . esc_html( $status ) . '</p><p>' . esc_html( $description ) . '</p><a href="' . esc_url( $transaction_url ) . '" target="_blank" rel="noopener noreferrer">' . esc_html( $callout ) . '</a>';
+				break;
+
+			case Fraud_Outcome_Status::REVIEW_ALLOWED:
+				$icon_url        = plugins_url( 'assets/images/icons/check-green.svg', WCPAY_PLUGIN_FILE );
+				$icon_alt        = __( 'Green check mark', 'woocommerce-payments' );
+				$status          = __( 'Held for review', 'woocommerce-payments' );
+				$description     = __( 'Need text to display once allowed.', 'woocommerce-payments' );
```

Comment:
Possible text:

> This transaction was held for review by your risk filters, and then the authorization was manually reviewed captured.

Reply:
Maybe

> This transaction was held for review by your risk filters, and the charge was manually approved after review.

?

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/5782/commits/4b4e2aa685151a4577427d1db4159f7d31c53ef3